src/firefly/js/ui/dynamic/EmbeddedPositionSearchPanel.jsx

```diff
@@ -351,30 +356,30 @@ function SearchSummary({request, targetKey, sizeKey, polygonKey, searchTypeKey,
 
     const target = request?.[targetKey];
     const polyCoords = request?.[polygonKey];
+    const coordPref = getPreference(MR_FIELD_HIPS_MOUSE_READOUT1, MR_EQJ2000_HMS);
     const coords = searchTypeValue === POLY_CHOICE_KEY ? polyCoords
-        : formatWorldPtToString(parseWorldPt(target)); // for cone, point, etc.
+        : formatWorldPtToString(parseWorldPt(target), getDcmOrHmsFromMR(coordPref), false); // for cone, point, etc.
```

Comment:
This is not quite right. If there is a resolved name you always want to show it instead of the ra/dec. I think you need to use `formatWorldPtToStringSimple` and modify that function to take the `dcmOrHms` parameter. I don't think the changes should be in `formatWorldPtToString`. 

That said I think they both should be change to accept a string or a point for the `wp` parameter. I will comment on that below